nixui/nix/lib.nix

```diff
@@ -1,6 +1,7 @@
 let
   inherit (import <nixpkgs> {}) pkgs lib;
   inherit (import <nixpkgs/nixos> {}) config;
+  modulesPath = <nixpkgs/nixos/modules>;
```

Comment:
i guess only `pkgs` should be taken from `<nixpkgs>` and the others from `<nixos>`
`nixos` and `nixpkgs` can be separate channels, for example, stable nixos + unstable nixpkgs

modulesPath is defined in [nixpkgs/nixos/lib/eval-config.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/eval-config.nix#L63)

evalModules is defined in [nixpkgs/lib/modules.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/modules.nix#L86)

Reply:
So it makes more sense for it to be the following?

```
  inherit (import <nixpkgs> {}) pkgs;
  inherit (import <nixpkgs/nixos> {}) config lib;
  modulesPath = <nixpkgs/nixos/modules>;
```

Reply:
```nix
let
  inherit (import <nixpkgs> {}) pkgs lib;
  inherit (import <nixos/nixos> {}) config;
  modulesPath = <nixos/nixos/modules>;
in
```

or


```nix
let
  inherit (import <nixpkgs> {}) pkgs lib;
  inherit (import <nixos> {}) config;
  modulesPath = <nixos/modules>;
in
```

Reply:
fixed by https://github.com/nix-gui/nix-gui/pull/210/commits/8c3089f4f133b647afe11e7de79d06de8659d9fd